src/vstart.sh

```diff
@@ -975,8 +989,16 @@ do_rgw()
     RGWSUDO=
     [ $CEPH_RGW_PORT -lt 1024 ] && RGWSUDO=sudo
     n=$(($CEPH_NUM_RGW - 1))
+    #extract rgw_frontends
+    local rgw_fe_tmp=`$CEPH_BIN/ceph-conf -c $conf_fn --name $VSTART_SEC rgw_frontends 2>/dev/null`
```

Comment:
i thought we agreed to pass `$rgw_frontend` and `$CEPH_RGW_PORT` directly instead of rereading them from ceph.conf. it looks like this would prevent you from changing the frontend type after running with `-n`